### ISSUE TITLE
08 - Bug Busters  - Audience_attention_Analysis

### ISSUE THREAD

**Issue by manish1610** [CodeBreakMIT/CodeBreak1.0#2]
**Before you start, please follow this format for your issue title**:  
8 BUG BUSTERS- Audience_attention_Analysis

## ℹ️ Project information
_Please complete all applicable._

**Project Name** :Audience_attention_Analysis

- **Short Project Description**: You can't hide being inattentive anymore, we'll always know.

**Team Code** : 8
**Team Name** : BUG BUSTERS
**Team Members**
 Manish Patel   https://github.com/manish1610
Vasu Garg        https://github.com/vasugargofficial
Raghav Bang   https://github.com/Raghav888
Vishal Kasa      https://github.com/vishalkasa 

- **Demo Link**:
**Video Link** : https://drive.google.com/file/d/1XiWgdsGwdwzHPRGvTOhKP3B1Q4lYC6RV/view?usp=drivesdk
**PPT Link** : https://docs.google.com/presentation/d/12Fy9tL4w8-8GSSSz99n4_ZwvHOSJj04Af3ZW2Z9o7pU/edit?usp=sharing
**Dependencies Link** :https://drive.google.com/file/d/1szuguiknEN1qmQUXFWHB1RwHNUP_3gpy/view?usp=sharing
- **Repository Link**: https://github.com/manish1610/Audience_attention_analysis.git
- **Labels**: Computer Vision,Data Science,Android Studio,Firebase,Python Libraries

## 🔥 Your Pitch

**Introduction** 

 The project is a tool that is able to give detail analysis of participant attention in a
gathering/conference/classroom/meetings via data visualization.

 It detects how many participants were attentive at different interval of time throughout
the meeting/gathering via detecting various features of audience.

**Applications** :

 In larger classrooms as well as conferences, it is not easy for teachers/speakers to come
to conclusion whether the audience was interested in their speech and detect at which
part of the speech most of audience were attentive,The application becomes handy at those places.

Similarly, it can be used in meetings to detect how many employers are taking the meetings
seriously.

 This project could also be used as a self guidance report for the speaker/teacher which he/she
could use to improve his content in speeches to make it more meaningful and increase
effectiveness.

 Also, a teacher can get recommendation letter from our system, which will contain detailed
analytical report of all past lectures of that faculty.

**Implementation**
Computer Vision
Data Science
Android App

Numerical Data is collected using computer vision and stored in excel sheet .

Then data visualization is done based on gathered data.

All visualized data is passed to server, and displayed to user on android app.

Our users are: Faculty, HOD’s, Speaker and team Manager.

 User need to login to see all the detailed analysed data.

## Technology Stack Used

Computer Vision – TensorFlow, Keras, NumPy and Pandas

 Data Science – matplotlib and Seaborn

 Android Studio

 Firebase

 Other Python libraries

## ✅ Checklist

**Before you post the issue**:
- [x] You have followed the issue title format.
- [x] You have mentioned the correct labels.
- [x] You have provided all the information correctly.
